musicbrainz-server/templates/default/sv-musicbrainz-server-run.erb

```diff
@@ -5,10 +5,14 @@ set -e
 cd mb_server
 umask 022
 exec \
-    env MUSICBRAINZ_USE_PROXY=1 starman -Ilib \
-        --preload-app \
-        -E deployment \
-        --listen musicbrainz-server.socket \
-        --workers <%= @variables[:nproc] || 5 %> \
-        --user www-data \
-        --group www-data
+    env MUSICBRAINZ_USE_PROXY=1 \
+        setuidgid www-data \
```

Comment:
Running the processes under another user/group isn't a builtin thing like it was with Starman, so I'm using `setuidgid` which is included in daemontools.